RecoTracker/MkFit/interface/MkFitClusterIndexToHit.h

```diff
@@ -0,0 +1,24 @@
+#ifndef RecoTracker_MkFit_MkFitClusterIndexToHit_h
+#define RecoTracker_MkFit_MkFitClusterIndexToHit_h
+
+#include <vector>
+
+class TrackingRecHit;
+
+class MkFitClusterIndexToHit {
+public:
+  MkFitClusterIndexToHit() = default;
+
+  std::vector<TrackingRecHit const *> &pixelHits() { return pixelHits_; }
+  std::vector<TrackingRecHit const *> const &pixelHits() const { return pixelHits_; }
+
+  std::vector<TrackingRecHit const *> &outerHits() { return outerHits_; }
+  std::vector<TrackingRecHit const *> const &outerHits() const { return outerHits_; }
+
+private:
+  // Indexed by cluster index
+  std::vector<TrackingRecHit const *> pixelHits_;
+  std::vector<TrackingRecHit const *> outerHits_;
```

Comment:
why do you use `outer` ? also OT has pixel modules.....

Reply:
So I've lost the proper nomenclature. What would you prefer that would cover both phase1 and phase2? `inner` and `outer`?

Reply:
in the rest of the code you use strip, if I'm not wrong
I would suggest to make use of an homogeneous naming scheme

Reply:
The class was split such that there are separate event products for pixel and strip. Nomenclature is now using "pixel" and "strip" everywhere, adaptation for phase2 will follow later.